triton_backend/all_models/llmapi/tensorrt_llm/1/helpers.py

```
@@ -177,6 +180,88 @@ def get_lora_request_from_request(request, batch_size=1, batch_index=0):
+    for name, spec in specs.items():
+        if not isinstance(spec, str) or ":" not in spec:
+            raise pb_utils.TritonModelException(
+                f"logits_post_processors['{name}'] must be an import spec "
+                f"of the form 'module.path:attribute', got {spec!r}")
+        module_name, _, attr_path = spec.partition(":")
+        try:
+            obj = importlib.import_module(module_name)
+        except ImportError as e:
+            raise pb_utils.TritonModelException(
+                f"logits_post_processors['{name}']: cannot import module "
+                f"'{module_name}': {e}")
```

Ruff (0.16.3)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)